update-script.js

```diff
@@ -217,11 +216,11 @@ const createAndUpdateRelatedRealmsRecords = async (client, bucket) => {
  * @return {Number} 0 for success, 1 for failure.
  */
 const main = async () => {
-  if (FX_RS_WRITER_USER === "" || FX_RS_WRITER_PASS === "") {
+  if (AUTHORIZATION === "") {
     console.error("No username or password set, quitting!");
     return 1;
   }
-  const secretString = `${FX_RS_WRITER_USER}:${FX_RS_WRITER_PASS}`;
+  const secretString = `${AUTHORIZATION}`;
```

Comment:
I would get rid of this `secretString` here, and just use `AUTHORIZATION` below:

```js
      headers: {
        Authorization: "Basic " + btoa(AUTHORIZATION)
      }
```